ui/hooks/gasFeeInput/useGasFeeErrors.js

```diff
@@ -131,6 +131,8 @@ const getMaxFeeWarning = (
 };
 
 const getBalanceError = (minimumCostInHexWei, transaction, ethBalance) => {
+  if (minimumCostInHexWei === undefined || ethBalance === undefined)
```

Comment:
What do you think about adding braces here? I believe that @darkwing wanted to add an ESLint rule around this not too long ago — [that PR seems to have been closed](https://github.com/MetaMask/metamask-extension/pull/12578) but I think it's still a good idea.

Reply:
Also does it make sense for this function to return false if there is nothing to return? Seems like we might want `null` or something similar.